linux/sockstat.go

```diff
@@ -0,0 +1,82 @@
+package linux
+
+import (
+	"io/ioutil"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+type SockStat struct {
```

Comment:
> Sorry for the delayed reply, I think we should also change Netstat to NetStat so it can actually follow the camel case naming rule. (users don't have to guess, since they don't have to know the under layer) :-)

Good point. Done in b2102064cce3eb0b82cbbc50d2347e34b17fa001.